scripts/build_industrial_production_per_country.py

```diff
@@ -296,7 +302,10 @@ def separate_basic_chemicals(demand, year):
 
     demand["Ammonia"] = 0.0
 
-    demand.loc[there, "Ammonia"] = ammonia.loc[there, str(year)]
+    year_to_use = min(max(year, 2018), 2022)
+    if year_to_use != year:
+        logger.info(f"Using data from {year_to_use} for ammonia production.")
+    demand.loc[there, "Ammonia"] = ammonia.loc[there, str(year_to_use)]
```

Comment:
```suggestion
    demand.loc[there, "Ammonia"] = ammonia.loc[there, year_to_use]
```
(if you do change suggested for `build_ammonia_production`)

Reply:
the columns are of type `str`, one could rename the columns to be of type `int` but this also creates one additional line of code. I would keep it as it is.